torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -13,6 +13,10 @@ struct UpgraderEntry {
 };
 
 const static std::unordered_map<std::string, std::vector<UpgraderEntry>> kOperatorVersionMap(
+    {{"aten::_test_serialization_subcmul",
```

Comment:
It's a bit unintuitive to add a test upgrader in the actual version map. Is the cost ok for us? cc: @iseeyuan

Reply:
Is it possible to emplace this test in another header file, and only include it when testing?

Reply:
@gmagogsfm  What do you think?

Reply:
It is not good practice to include test-only code in prod code path. One common pattern is to have test-only utilities that allow test cases to register a different set of data (upgraders in this case) so that the tests that cover infrastructure for upgrader support is completely decoupled from the existence of any prod upgraders.